cmd/data-node/commands/unsafe_reset_all.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 Gobalsky Labs Limited
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at https://www.mariadb.com/bsl11.
+//
+// Change Date: 18 months from the later of the date of the first publicly
+// available Distribution of this version of the repository, and 25 June 2022.
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by version 3 or later of the GNU General
+// Public License.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"code.vegaprotocol.io/vega/datanode/config"
+	"code.vegaprotocol.io/vega/datanode/sqlstore"
+	"code.vegaprotocol.io/vega/logging"
+	"code.vegaprotocol.io/vega/paths"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type UnsafeResetAllCmd struct {
+	config.VegaHomeFlag
+	*config.Config
+}
+
+//nolint:unparam
+func (cmd *UnsafeResetAllCmd) Execute(_ []string) error {
+	log := logging.NewLoggerFromConfig(
+		logging.NewDefaultConfig(),
+	)
+	defer log.AtExit()
+
```

Comment:
I'm not sure if you want to add a warning and prompt for confirmation that they really want to do this, while they have to specifically want to run the command, usually when you are about to wipe all your data, you are warned about it as a precaution. Just a suggestion though.

Reply:
Yeah, I based this on what the action does for core, it doesn't prompt for confirmation, I believe the reasoning here is that the name of the command itself is sufficient warning, which is reasonable